gr-analog/lib/fastnoise_source_impl.cc

```diff
@@ -76,7 +77,8 @@ fastnoise_source_impl<gr_complex>::fastnoise_source_impl(noise_type_t type,
                  io_signature::make(0, 0, 0),
                  io_signature::make(1, 1, sizeof(gr_complex))),
       d_type(type),
-      d_ampl(ampl / sqrtf(2.0f))
+      d_ampl(ampl / sqrtf(2.0f)),
```

Comment:
really doesn't matter at the rate at which this will be called, but as something I've learned: even GCC 9.3 with `-O3` will implement that as a division by a constant  ([godbolt](https://gcc.godbolt.org/z/pFzmYY)); `ampl * sqrtf(0.5f)` would be more efficient.

Reply:
`sqrt(2.0)` to me is a known formula, thus makes it easier to understand where that magic number came from. But you prefer multiplying by `sqrt(0.5)`)

Reply:
yeah, you're right, as said, this is really not the place to replace a clear code by a slightly more performant one :)